bbc/simorgh
# WSTEAM1-313: Unsupported PS embeds on .amp

Resolves JIRA [WSTEAM1-313](https://jira.dev.bbc.co.uk/browse/WSTEAM1-313) - Embeds: Optimo Telescope embeds (.amp support only)

Overall changes
======
Show 'link to Canonical' for Telescope embeds on PS .amp pages.

Note there is a BFF PR `WSTEAM1-313-handle-unsupported-embeds` that will be merged and deployed to live after this PR. This BFF PR amends the data schema for the UploaderEmbed. There is a comment on a section of code that will be removed once both PRs are on live.

Code changes
======
- Create UnsupportedEmbed component to handle Telescope Embed block type on .amp. (Will also handle Optimo Sport embeds by default - though tests for this will be handled in a follow up ticket)
- Move Uploader component inside Embed folder
- Add Telescope embed fixture data
- Add unit tests

Testing
======
Locally
1. Pull down this PR and use `yarn dev` to run Simorgh locally
2. Visit Telescope uploader page: http://localhost:7080/news/articles/c77z1dexexzo.amp
3. See 'error message' including link to cananoical
4. Regression test UGC uploader component here Canonical: http://localhost:7080/pidgin/articles/cwl08rd38l6o and here .amp http://localhost:7080/pidgin/articles/cwl08rd38l6o.amp

On Test
Find testing URLs and a note on the URL Redirect in a Confluence Link on the [Ticket](https://jira.dev.bbc.co.uk/browse/WSTEAM1-313).

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
